No signature of method: java.lang.String.uncapitalize()

Hello,

When I use dexcount 0.7.0 with Android Gradle Plugin 2.3.1 and Gradle 3.3 I got the following exception:
```
Caused by: groovy.lang.MissingMethodException: No signature of method: java.lang.String.uncapitalize() is applicable for argument types: () values: []
Possible solutions: capitalize()
        at com.getkeepsafe.dexcount.DexMethodCountPlugin$Provider.createTask(DexMethodCountPlugin.groovy:177)
        at com.getkeepsafe.dexcount.DexMethodCountPlugin$LegacyProvider$_applyToApkVariant_closure2.doCall(DexMethodCountPlugin.groovy:217)
        at com.getkeepsafe.dexcount.DexMethodCountPlugin$LegacyProvider.applyToApkVariant(DexMethodCountPlugin.groovy:215)
        at com.getkeepsafe.dexcount.DexMethodCountPlugin$LegacyProvider.applyToApplicationVariant(DexMethodCountPlugin.groovy:197)
        at org.gradle.internal.metaobject.BeanDynamicObject$MetaClassAdapter.invokeMethod(BeanDynamicObject.java:382)
        at org.gradle.internal.metaobject.BeanDynamicObject.invokeMethod(BeanDynamicObject.java:170)
        at org.gradle.internal.metaobject.ConfigureDelegate.invokeMethod(ConfigureDelegate.java:80)
        at com.getkeepsafe.dexcount.DexMethodCountPlugin$Provider$_apply_closure1.doCall(DexMethodCountPlugin.groovy:134)
...
```

The reasons are:
1. Method uncapitalize() is used by [DexMethodCountPlugin.Provider](https://github.com/KeepSafe/dexcount-gradle-plugin/blob/91ec56bc5b9ce6397bc22e3bf9940c45df3283f7/src/main/groovy/com/getkeepsafe/dexcount/DexMethodCountPlugin.groovy#L177).
2. Method [CharSequence.uncapitalize()](http://docs.groovy-lang.org/latest/html/groovy-jdk/java/lang/CharSequence.html#uncapitalize()) is supported since Groovy 2.4.8.
3. Version of Groovy bundled with Gradle 3.3 is 2.4.7. Only in [Gradle 3.5 this version is upgraded to 2.4.10](https://docs.gradle.org/current/release-notes.html#groovy-upgraded-from-2.4.7-to-2.4.10).

The hot fix solution is to declare uncapitalize method:
```groovy
String.metaClass.uncapitalize = {
    if (delegate.length() <= 1) return delegate.toLowerCase()
    return delegate[0].toLowerCase() + delegate[1..-1]
}
```

--- a/src/main/groovy/com/getkeepsafe/dexcount/DexMethodCountPlugin.groovy
+++ b/src/main/groovy/com/getkeepsafe/dexcount/DexMethodCountPlugin.groovy
@@ -165,16 +165,18 @@ class DexMethodCountPlugin implements Plugin<Project> {
                 @ClosureParams(value = SimpleType, options = ['com.getkeepsafe.dexcount.DexMethodCountTask']) Closure applyInputConfiguration) {
             def slug = variant.name.capitalize()
             def path = "${project.buildDir}/outputs/dexcount/${variant.name}"
+            def outputName = variant.name
             if (variant.outputs.size() > 1) {
                 assert output != null
                 slug += output.name.capitalize()
                 path += "/${output.name}"
+                outputName = output.name
             }
 
             def task = project.tasks.create("count${slug}DexMethods", DexMethodCountTask)
             task.description = "Outputs dex method count for ${variant.name}."
             task.group = 'Reporting'
-            task.variantOutputName = slug.uncapitalize()
+            task.variantOutputName = outputName
             task.mappingFile = variant.mappingFile
             task.outputFile = project.file(path + ext.format.extension)
             task.summaryFile = project.file(path + '.csv')

